[8.4](backport #2463) clarificaton on the comment for root permissions on Agent (#2638)
elastic/observability-docs: docs/en/ingest-management/beats-agent-comparison.asciidoc

|Run without root on host
|{y}
|{n}

> |Run without root on host
> |{y}

Reviewer: ```suggestion
```

> |{n}

Reviewer: ```suggestion
```

|Run without root on host
|{y}
|{n}

> <<<<<<< HEAD
> |{y}
> |{fleet}-managed {agent}s require root permission, in particular for {endpoint-cloud-sec}. Standalone {agent}s and {beats} do not.
> =======
> |{n}
> |{fleet}-managed {agent}s require root permission, in particular for {elastic-defend}. Standalone {agent}s also require root permissions. {beats} requires root permission only if it's configured to capture data that requires that level of permission.
> >>>>>>> 016a809c (clarificaton on the comment for root permissions on Agent (#2463))

Reviewer: ```suggestion
|Run without root on host
|{y}
|{n}
|{n}
|{fleet}-managed {agent}s require root permission, in particular for {elastic-defend}. Standalone {agent}s also require root permissions. {beats} requires root permission only if it's configured to capture data that requires that level of permission.
```
